src/pages/dev/general-message-passing/overview.mdx

```diff
@@ -21,14 +21,52 @@ With GMP you can:
 #### At the source chain
 
 1. User (dApp) calls a `callContract` (or `callContractWithToken`) function on the Axelar Gateway contract to initiate a call. Once the call is initiated, the user can see its status at https://axelarscan.io/gmp/[txHash] or programmatically track it via the [AxelarJS SDK](/dev/axelarjs-sdk/tx-status-query-recovery#query-transaction-status-by-txhash).
-2. User prepays the relayer gas fee on the source chain to Axelar’s Gas Services contract.
+2. The user prepays the relayer gas fee on the source chain to Axelar's Gas Services contract.
 3. The call enters the Axelar Gateway from the source chain.
 
 #### At the Axelar network
-4. Axelar network confirms the call and converts the paid gas from the source chain’s native token into the destination chain’s native token.  
+4. The Axelar network confirms the call and converts the paid gas from the source chain's native token into the destination chain's native token.  
 
 #### At the destination chain
 5. The call is approved and emerges from the Axelar Gateway on the destination chain.
```

Comment:
Worth adding a note here on "approved"? Just could be a brief sentence that approved means axelar validators coming to consensus and they generate a key to sign off on the approval